applications/HDF5Application/python_scripts/partitioned_single_mesh_primal_input_process.py

```diff
@@ -11,18 +11,27 @@ def Factory(settings, Model):
                 "model_part_name" : "MainModelPart",
                 "file_settings" : {},
                 "nodal_results_settings" : {},
-                "time_tag_precision" : 4
+                "element_results_settings" : {},
+                "time_tag_precision" : 4,
+                "file_name": "DEFAULT_NAME"
             }
             """)
     settings = settings["Parameters"].Clone()
     settings.ValidateAndAssignDefaults(default_settings)
     model_part = Model[settings["model_part_name"].GetString()]
     hdf5_file_factory = hdf5_io.HDF5ParallelFileFactory(settings["file_settings"])
     nodal_results_input = hdf5_io.PrimalBossakInput(settings["nodal_results_settings"])
+    element_results_input = hdf5_io.ElementResultsInput(settings["element_results_settings"])
     input_time_settings = KratosMultiphysics.Parameters("""{}""")
     input_time_settings.AddEmptyValue("time_tag_precision")
     input_time_settings["time_tag_precision"].SetInt(settings["time_tag_precision"].GetInt())
+    input_time_settings.AddEmptyValue("file_name")
```

Comment:
Can't we replace DEFAULT_NAME with the model part name like you did in the other files? This would eliminate this if statement.